tremor-tool/src/cli.yaml

```diff
@@ -1,4 +1,4 @@
-name: tremor-cli
+name: tremor-tool
 version: "1.0"
```

Comment:
does the version here change anything for the version determination?

Reply:
nope it has no function, good call we might as well kill it